src/Core/Core.idr

```diff
@@ -418,70 +418,37 @@ export
 %inline
 coreLift : IO a -> Core a
 coreLift op = MkCore (do op' <- op
-                         pure (Right op'))
-
-{- Monad, Applicative, Traversable are specialised by hand for Core.
```

Comment:
Are we there yet ? I know that the partial evaluator is a bit broken. But, on the other hand, there is machinery that transforms parametric method calls to specialised versions. The question is: does it work well ? Better ask @edwinb :)

Reply:
On testing it went from make of 114 seconds to 115 seconds. Separating this commit into #856)